doc/03-Configuration.md

```diff
@@ -34,65 +29,24 @@ port:
 
 `443,5665-5669`
 
-Scan jobs can be executed using the `icingacli x509 scan` CLI command. The `--job` option is used to specify the scan
-job which should be run:
-
-```
-icingacli x509 scan --job lan
-```
-
-## Scheduling Jobs
-
-Each job may specify a `cron` compatible `schedule` to run periodically at the given interval. The `cron` format is as
-follows:
-
-```
-*    *    *    *    *
--    -    -    -    -
-|    |    |    |    |
-|    |    |    |    |
-|    |    |    |    +----- day of week (0 - 6) (Sunday to Saturday)
-|    |    |    +---------- month (1 - 12)
-|    |    +--------------- day of month (1 - 31)
-|    +-------------------- hour (0 - 23)
-+------------------------- minute (0 - 59)
-```
-
-Example definitions:
-
-Description                                                 | Definition
-------------------------------------------------------------| ----------
-Run once a year at midnight of 1 January                    | 0 0 1 1 *
-Run once a month at midnight of the first day of the month  | 0 0 1 * *
-Run once a week at midnight on Sunday morning               | 0 0 * * 0
-Run once a day at midnight                                  | 0 0 * * *
-Run once an hour at the beginning of the hour               | 0 * * * *
-
-Jobs are executed on CLI with the `jobs` command:
-
-```
-icingacli x509 jobs run
-```
+Additionally, each job may also exclude specific **hosts** and **IP** addresses from scan. These hosts won't be scanned
+when you run the [scan](04-Scanning.md#scan-command) or [jobs](04-Scanning.md#scheduling-jobs) command. Excluding an entire network and specifying IP addresses in CIDR
+format will not work. You must specify concrete **IP** and **host CN** separated with commas, e.g:
```

Comment:
Shouldn't this read `You must specify concrete **IP**s or **host CN**s separated with commas`?